{"uuid":"ab38c814-8059-4ac5-9116-88bc0ce03abb","historyId":"7f04514ab2cc505bc0ee6f895c5d8eb6","fullName":"com.apiFramework.tests.crud.TestCreateBooking.testCreateBookingPOST_POSITIVE","labels":[{"name":"package","value":"com.apiFramework.tests.crud.TestCreateBooking"},{"name":"testClass","value":"com.apiFramework.tests.crud.TestCreateBooking"},{"name":"testMethod","value":"testCreateBookingPOST_POSITIVE"},{"name":"parentSuite","value":"All Test Suite"},{"name":"suite","value":"/Users/dheerajv/IdeaProjects/APIAutomationFramework1"},{"name":"subSuite","value":"com.apiFramework.tests.crud.TestCreateBooking"},{"name":"host","value":"dheerajv-mac"},{"name":"thread","value":"42575@dheerajv-mac.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"owner","value":"Dheeraj"}],"links":[],"name":"testCreateBookingPOST_POSITIVE","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"expected [200] but found [Optional.of(200)]","trace":"java.lang.AssertionError: expected [200] but found [Optional.of(200)]\n\tat org.testng.Assert.fail(Assert.java:111)\n\tat org.testng.Assert.failNotEquals(Assert.java:1578)\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:150)\n\tat org.testng.Assert.assertEquals(Assert.java:132)\n\tat org.testng.Assert.assertEquals(Assert.java:644)\n\tat com.apiFramework.asserts.AssertActions.verifyStatusCode(AssertActions.java:19)\n\tat com.apiFramework.tests.crud.TestCreateBooking.testCreateBookingPOST_POSITIVE(TestCreateBooking.java:30)\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:580)\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1597)\n\tat org.testng.TestRunner.privateRun(TestRunner.java:739)\n\tat org.testng.TestRunner.run(TestRunner.java:614)\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:421)\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:312)\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1208)\n\tat org.testng.TestNG.runSuites(TestNG.java:1112)\n\tat org.testng.TestNG.run(TestNG.java:1079)\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)\n"},"stage":"finished","description":"TC#1 - Verify that the Booking can be created","steps":[],"attachments":[],"parameters":[],"start":1755057708196,"stop":1755057710604}